src/transformers/trainer.py

```diff
@@ -704,6 +705,7 @@ def get_train_dataloader(self) -> DataLoader:
             drop_last=self.args.dataloader_drop_last,
             num_workers=self.args.dataloader_num_workers,
             pin_memory=self.args.dataloader_pin_memory,
+            worker_init_fn=seed_worker,
```

Comment:
Is this something that should always be enabled, or should it check if `self.args.seed` is set and only set it in case it's not `None`?

Reply:
I think it doesn't hurt doing it all the time.